django/contrib/gis/gdal/srs.py

```diff
@@ -30,9 +30,13 @@
 
 from django.contrib.gis.gdal.base import GDALBase
 from django.contrib.gis.gdal.error import SRSException
+from django.contrib.gis.gdal.libgdal import GDAL_VERSION
 from django.contrib.gis.gdal.prototypes import srs as capi
 from django.utils.encoding import force_bytes, force_str
 
+OAMS_TRADITIONAL_GIS_ORDER = 0
+OAMS_AUTHORITY_COMPLIANT = 1
```

Comment:
I think we should use `IntEnum` for this and expect its instances as `axis_order` argument.